wiki/Game_mode/osu!/id.md

```diff
@@ -0,0 +1,291 @@
+<!-- TODO: strange descriptions in many places and unhelpful order if you're trying to see what this game mode is all about. -->
+
+# osu! (mode permainan)
+
+*Untuk kegunaan lain, lihat [osu! (disambiguasi)](/wiki/Disambiguasi/osu!).*
+
+![Tampilan permainan osu!](/wiki/shared/osu-gameplay.jpg "Antarmuka osu!")
+
+Mode permainan pertama yang namanya didasarkan dari seri ouendan.
+
+## Apa itu "Ouendan"?
+
+*[Osu! Tatakae! Ouendan](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan "Wikipedia")* adalah permainan ritme musik untuk *Nintendo DS*. Alur pada permainan ini terdiri dari tiga elemen: mengetuk lingkaran pada layar sentuh, menyeret bola melintasi jalur dan memutar *spinner* dengan sangat cepat. Semua elemen ini diatur waktunya oleh (cover dari) lagu-lagu Jepang populer, Permainan tersebut akan terlihat seperti ini pada sistem *Nintento DS*:
```

Comment:
```suggestion
*[Osu! Tatakae! Ouendan](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan "Wikipedia")* adalah permainan ritme musik untuk *Nintendo DS*. Alur pada permainan ini terdiri dari tiga elemen: mengetuk lingkaran pada layar sentuh, menyeret bola melintasi jalur dan memutar *spinner* dengan sangat cepat. Semua elemen ini diatur waktunya oleh (cover dari) lagu-lagu Jepang populer, Permainan tersebut akan terlihat seperti ini pada sistem *Nintendo DS*:
```

Reply:
you made a typo here-

Reply:
oh yes, i didn't realize that
fixed now